modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/VerticalNav.js

```diff
@@ -42,6 +43,18 @@ export default function VerticalNav({
 					key={item.id}
 				>
 					{item.label}
+
+					{item.icons &&
+						item.icons.map((icon) => {
```

Comment:
[nit] We can use optional chaining here, to simplify a bit, `item.icons?.map(...)`

Reply:
done